lib/exercises.rb

```diff
@@ -1,19 +1,46 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 Since you're sorting the strings, the time complexity is O(n + m log m) where m is the length of the words, but if you assume the strings are limited in size, you can say it's O(n).